tests/zfs-tests/tests/functional/mmp/mmp_active_import.ksh

```diff
@@ -60,10 +60,10 @@ log_must is_pool_imported $MMP_POOL "-d $MMP_DIR"
 
 # 3. Verify 'zpool import [-f] $MMP_POOL' cannot import the pool.
 MMP_IMPORTED_MSG="Cannot import '$MMP_POOL': pool is imported"
-log_must try_pool_import $MMP_POOL "-d $MMP_DIR" $MMP_IMPORTED_MSG
+log_must try_pool_import $MMP_POOL "-d $MMP_DIR" "$MMP_IMPORTED_MSG"
 for i in {1..10}; do
 	log_must pgrep ztest >/dev/null
```

Comment:
Interestingly we saw a [test failure](http://build.zfsonlinux.org/builders/Ubuntu%2014.04%20i686%20%28TEST%29/builds/3897/steps/shell_8/logs/log) for this test case.  Based on the log I think we failed this `log_must pgrep ztest` and the error message was send to `/dev/null`.  Let's drop this check entirely.  It was only added to allow us to check for the case when ztest unexpectedly terminates early, and it doesn't even do that well due to the `>/dev/null`.

Reply:
Why do you think that is was caused the test to fail?  I looked at the log and it's not clear to me what happened there.

Reply:
That's exactly why I think this line is responsible.  If the failure happened anywhere else in this loop we would have gotten the failure output from `log_must`.  I think we should simply remove it.  If it's the case that `ztest` exited early then we'll catch that when the `zpool import` unexpected succeeds.

Reply:
I see.  I'll remove it.